ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbar.java

```diff
@@ -290,6 +290,7 @@ private void setIconSource(ReadableMap source, IconControllerListener controller
               .setOldController(holder.getController())
               .build();
       holder.setController(controller);
+      holder.getTopLevelDrawable().setVisible(true, true);
```

Comment:
Can `getTopLevelDrawable` return `null`?

What is `true, true`?

Why does this work? Can you add a comment explaining what the line does?